home.admin/config.scripts/cl.install.sh

```diff
@@ -42,7 +42,7 @@ function installDependencies() {
     libsqlite3-dev python3 net-tools zlib1g-dev libsodium-dev \
     gettext
   # additional requirements
-  sudo apt-get install -y postgresql libpq-dev
+  sudo apt-get install -y postgresql libpq-dev psutil
```

Comment:
There is no Debian package called `psutil`. This is a pip package which is installed as a dependency of other packages.
```suggestion
  sudo apt-get install -y postgresql libpq-dev
```